app/assets/stylesheets/companion_window.css

```diff
@@ -308,7 +308,7 @@
   .vert-tabs,
   .left-drawer,
   .modal-components-popover,
-  .sul-embed-treegrid,
+  .sul-embed-treegrid .folder-title,
```

Comment:
What about the file icons? Don't they also want this style?

Reply:
I checked a bunch on main and it looks like changing the font-size in no way changes the filename icon size